xgcm/grid.py

```diff
@@ -168,7 +203,10 @@ def __init__(self, ds, axis_name, periodic=True, default_shifts={}, coords=None)
 
     def __repr__(self):
         is_periodic = "periodic" if self._periodic else "not periodic"
-        summary = ["<xgcm.Axis '%s' %s>" % (self.name, is_periodic)]
+        summary = [
+            "<xgcm.Axis '%s' (%s, boundary=%r)>"
+            % (self.name, is_periodic, self.boundary)
+        ]
         summary.append("Axis Coodinates:")
```

Comment:
Just noticed this: "Coodinates", should probably be "Coordinates. Not sure if we want to do this in a seperate PR?